aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -78,30 +76,26 @@ const Users = () => {
                 greater detail behind this summary information.
               `);
             }}
-          >
-            <CardBody gap="xsmall" justify="between">
-              <Box flex={false}>
-                <Box align="center" direction="row" gap="xsmall">
-                  <Box
-                    background={
-                      datum.status === 'Online' ? 'brand' : 'text-weak'
-                    }
-                    pad="xsmall"
-                    round
-                  />
-                  <Text color="text-strong">{datum.status}</Text>
-                </Box>
-                <Text color="text-strong" size="large" weight="bold">
-                  {datum.name}
-                </Text>
-                <Text color="text-strong">{datum.location}</Text>
+            icon={
+              <Box align="center" direction="row" gap="xsmall">
+                <Box
+                  background={datum.status === 'Online' ? 'brand' : 'text-weak'}
+                  pad="xsmall"
+                  round
+                />
+                <Text color="text-strong">{datum.status}</Text>
               </Box>
+            }
+            title={datum.name}
+            subtitle={datum.location}
+            actions={
               <Box>
                 <Text size="small">Role</Text>
                 <Text color="text-strong">{datum.role}</Text>
               </Box>
-            </CardBody>
-          </Card>
+            }
```

Comment:
This seems like we need to revisit how our Card is built up. "actions" doesn't seem like the right name/place for this type of content. Would treating this content as `children` make more sense? Children could then be inserted as child of CardBody?

Reply:
Could call, I'll take a look at that. I was feeling the same.